web/modules/custom/joinup_seo/src/Plugin/simple_sitemap/UrlGenerator/JoinupUrlGeneratorBase.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
This is formatted according to the PSR-12 standard. In the Drupal standard there should be spaces around the `=` symbol.